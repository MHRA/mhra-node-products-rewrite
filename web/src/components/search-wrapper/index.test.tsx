import renderer from 'react-test-renderer';
import React from 'react';
import SearchWrapper from './index';

describe(SearchWrapper, () => {
  it('should render', () => {
    const component = renderer
      .create(
        <SearchWrapper initialSearchValue="initial value">
          <div>Child contents</div>
        </SearchWrapper>,
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
