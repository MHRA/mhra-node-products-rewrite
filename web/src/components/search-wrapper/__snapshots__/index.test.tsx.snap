// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchWrapper should render 1`] = `
<div
  className="sc-papXJ fXFsBZ"
>
  <section
    className="search"
  >
    <h2
      className="sc-jqUVSM jRVGCC"
    >
      Search box
    </h2>
    <section
      className="sc-eCYdqJ kNJCcP"
    >
      <h3
        className="sc-jSMfEi iEueme"
      >
        Search by text
      </h3>
      <form
        aria-label="Search for medicines"
        onSubmit={[Function]}
        role="search"
      >
        <label
          htmlFor="search"
        >
          Enter a product, active substance, or PL number:
        </label>
        <div
          className="searchbar"
        >
          <input
            id="search"
            onBlur={[Function]}
            onChange={[Function]}
            type="search"
            value=""
          />
          <input
            className="sc-hKMtZM jqQRbd"
            type="submit"
            value="Search"
          />
        </div>
      </form>
    </section>
    <nav
      className="sc-bczRLJ hrwWmM"
    >
      <p
        className="horizontal"
      >
        or find by active substance:
      </p>
      <ul
        className="horizontal"
      >
        <li
          className=""
        >
          <a
            href="/substance-index?letter=A"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            A
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=B"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            B
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=C"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            C
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=D"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            D
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=E"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            E
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=F"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            F
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=G"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            G
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=H"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            H
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=I"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            I
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=J"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            J
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=K"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            K
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=L"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            L
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=M"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            M
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=N"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            N
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=O"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            O
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=P"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            P
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=Q"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Q
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=R"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            R
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=S"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            S
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=T"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            T
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=U"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            U
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=V"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            V
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=W"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            W
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=X"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            X
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=Y"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Y
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=Z"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Z
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=0"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            0
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=1"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            1
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=2"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            2
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=3"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            3
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=4"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            4
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=5"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            5
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=6"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            6
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=7"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            7
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=8"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            8
             
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/substance-index?letter=9"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            9
             
          </a>
        </li>
      </ul>
    </nav>
  </section>
  <section
    className="sc-gKXOVf gqXioK"
  >
    <header
      aria-label="Yellow card"
    >
      <h2
        className="sc-ftvSup jmUtve"
      >
        Yellow Card
      </h2>
    </header>
    <div
      className="action-bar"
    >
      <div
        className="sc-iBkjds eHPKFW"
        role="img"
      >
        <div>
          <svg
            fill="none"
            height={31}
            role="img"
            viewBox="0 0 202 31"
            width={202}
          >
            <title>
              Yellow Card
            </title>
            <path
              d="M58.322 3.947l-8.94 14.267v9.346h-4.723v-9.244l-8.923-14.37h5.433l3.338 5.6a83.882 83.882 0 012.564 4.623 83.94 83.94 0 012.565-4.623l3.32-5.6h5.365zm13.023 14.775c.002.496-.02.99-.067 1.484H60.68a3.297 3.297 0 001.25 2.564c.69.532 1.546.806 2.416.774 1.303 0 2.314-.416 3.035-1.247l3.306 2.766a6.941 6.941 0 01-2.784 2.12 8.831 8.831 0 01-3.556.748c-2.544 0-4.568-.832-6.072-2.497-1.505-1.665-2.259-3.902-2.261-6.712 0-2.698.748-4.914 2.243-6.647a7.145 7.145 0 015.651-2.597 6.53 6.53 0 015.497 2.734c1.292 1.708 1.939 3.878 1.939 6.51zm-4.525-2.024a3.702 3.702 0 00-.851-2.193 3.023 3.023 0 00-2.353-.979 2.555 2.555 0 00-2.013.894 3.953 3.953 0 00-.922 2.278h6.14zM78.498 27.56h-4.656V5.328l4.656-2.324V27.56zm8.162 0h-4.65V5.328l4.655-2.324-.003 24.556zm18.45-8.838c0 2.699-.758 4.908-2.275 6.627a7.676 7.676 0 01-8.892 1.907 7.68 7.68 0 01-2.595-1.907c-1.516-1.72-2.275-3.928-2.275-6.627 0-2.698.759-4.915 2.275-6.65a7.65 7.65 0 0111.501 0c1.507 1.737 2.261 3.954 2.261 6.65zm-4.723 0a6.295 6.295 0 00-.84-3.44 2.748 2.748 0 00-2.473-1.318 2.717 2.717 0 00-2.437 1.286c-.551.864-.827 2.017-.827 3.458 0 1.441.27 2.566.81 3.374a2.733 2.733 0 002.472 1.349 2.826 2.826 0 002.497-1.452c.565-.99.838-2.118.788-3.257h.01zm28.912-8.872l-5.097 17.71h-3.886l-1.766-6.951c-.497-1.978-.812-3.373-.946-4.183-.135.81-.449 2.205-.944 4.183l-1.766 6.951h-3.885l-5.055-17.71h4.723l1.55 6.545c.406 1.686.687 3.035.845 4.048.134-.81.448-2.16.943-4.048l1.766-6.546h3.847l1.766 6.546c.494 1.884.81 3.233.946 4.048.156-1.015.436-2.365.841-4.048l1.554-6.546h4.564zm25.206-1.046l-4.383 2.024c-.9-1.71-2.339-2.565-4.317-2.565a4.762 4.762 0 00-4.083 2.06c-.989 1.37-1.484 3.18-1.484 5.429 0 2.183.495 3.96 1.484 5.33a4.764 4.764 0 004.115 2.158c1.884 0 3.424-1 4.62-3.002l3.981 2.363c-1.731 3.553-4.643 5.33-8.735 5.33-3.106 0-5.602-1.102-7.489-3.306-1.886-2.204-2.828-5.162-2.826-8.873 0-3.596.978-6.524 2.932-8.785a9.536 9.536 0 017.552-3.373c4.069 0 6.947 1.736 8.633 5.21zm17.036 18.756h-4.655v-1.18c-.989 1.032-2.316 1.549-3.981 1.551a6.872 6.872 0 01-4.553-1.55c-1.371-1.148-2.057-2.738-2.06-4.773 0-1.884.699-3.357 2.095-4.419 1.281-.968 2.979-1.451 5.094-1.451a7.53 7.53 0 013.405.706v-.84a2.172 2.172 0 00-.657-1.622 2.58 2.58 0 00-1.872-.64 10.069 10.069 0 00-4.857 1.18l-1.788-3.239a13.106 13.106 0 016.645-1.787c2.178 0 3.92.545 5.227 1.636 1.307 1.09 1.96 2.613 1.957 4.57V27.56zm-4.655-5.196V20.14c-.788-.495-1.865-.742-3.239-.742-1.8 0-2.699.742-2.699 2.225 0 1.576.899 2.362 2.699 2.36 1.509 0 2.589-.54 3.239-1.618zm19.194-11.639l-1.215 4.557a3.91 3.91 0 00-2.699-1.148c-.989 0-1.72.259-2.193.777-.537.604-.809 1.628-.809 3.07v9.58h-4.656V9.848h4.656v1.18a4.851 4.851 0 013.645-1.55c1.392 0 2.482.415 3.271 1.246zM202 27.56h-4.652v-1.113a5.542 5.542 0 01-3.744 1.484 5.927 5.927 0 01-4.861-2.26c-1.349-1.649-2.024-4.044-2.024-7.185 0-2.699.608-4.875 1.823-6.528 1.215-1.653 2.902-2.477 5.062-2.472a5.357 5.357 0 013.744 1.384V5.321L202 3.004V27.56zm-4.652-5.5v-6.68a3.675 3.675 0 00-3.14-1.514c-1.844 0-2.766 1.505-2.766 4.517 0 3.44 1.013 5.16 3.038 5.16a3.493 3.493 0 002.868-1.483zM17.196 1.595a1.595 1.595 0 11-3.19 0 1.595 1.595 0 013.19 0zM23.59 3.12a1.595 1.595 0 11-3.19 0 1.595 1.595 0 013.19 0zm5.143 4.467a1.595 1.595 0 11-3.189 0 1.595 1.595 0 013.19 0zm2.44 6.36a1.594 1.594 0 11-3.19 0 1.594 1.594 0 013.19 0zm-2.331 8.128a1.595 1.595 0 100-3.19 1.595 1.595 0 000 3.19zm-3.815 5.65a1.595 1.595 0 100-3.19 1.595 1.595 0 000 3.19zm-4.422 1.615a1.595 1.595 0 11-3.19 0 1.595 1.595 0 013.19 0zm-6.579.079a1.596 1.596 0 11-3.192-.03 1.596 1.596 0 013.192.03zm-6.081-3.125a1.595 1.595 0 11-3.19.01 1.595 1.595 0 013.19-.01zm-3.901-5.58a1.595 1.595 0 11-3.19 0 1.595 1.595 0 013.19 0zM3.19 14.19A1.595 1.595 0 110 14.2a1.595 1.595 0 013.19-.01zM5.52 7.8a1.595 1.595 0 11-3.19 0 1.595 1.595 0 013.19 0zm5.063-4.565a1.595 1.595 0 11-3.19 0 1.595 1.595 0 013.19 0zm6.449 3.035a1.432 1.432 0 11-2.863.01 1.432 1.432 0 012.863-.01zm-4.407 1.098a1.432 1.432 0 11-2.864.01 1.432 1.432 0 012.864-.01zm8.67-.074a1.431 1.431 0 11-2.862.01 1.431 1.431 0 012.862-.01zm3.434 2.981a1.43 1.43 0 11-2.861.01 1.43 1.43 0 012.86-.01zm1.629 4.238a1.431 1.431 0 11-2.862 0 1.431 1.431 0 012.862 0zm-8.973 1.157a1.774 1.774 0 11-3.548.001 1.774 1.774 0 013.548 0zm8.481 3.21a1.431 1.431 0 11-2.862 0 1.431 1.431 0 012.862 0zm-2.519 3.767a1.43 1.43 0 11-2.862.01 1.43 1.43 0 012.862-.01zm-4.039 2.141a1.43 1.43 0 11-2.862 0 1.43 1.43 0 012.862 0zm-4.358.04a1.431 1.431 0 11-2.863.01 1.431 1.431 0 012.863-.01zm-4.084-2.072a1.431 1.431 0 11-2.863.01 1.431 1.431 0 012.863-.01zM8.258 19.03a1.43 1.43 0 11-2.861.01 1.43 1.43 0 012.86-.01zm-.572-4.348a1.431 1.431 0 11-2.862 0 1.431 1.431 0 012.862 0zm1.56-4.272a1.431 1.431 0 11-2.862 0 1.431 1.431 0 012.862 0zm7.612.402a1.258 1.258 0 11-2.515-.02 1.258 1.258 0 012.515.02zm3.772 1.808a1.257 1.257 0 11-2.514.01 1.257 1.257 0 012.514-.01zm.963 4.06a1.257 1.257 0 11-2.514.009 1.257 1.257 0 012.514-.01zm-2.603 3.373a1.257 1.257 0 11-2.515 0 1.257 1.257 0 012.515 0zm-4.179.015a1.256 1.256 0 11-2.512.01 1.256 1.256 0 012.512-.01zm-2.673-3.304a1.257 1.257 0 11-2.514 0 1.257 1.257 0 012.514 0zm.895-4.079a1.258 1.258 0 11-2.516.01 1.258 1.258 0 012.516-.01z"
              fill="#010101"
            />
          </svg>
        </div>
      </div>
      <p>
        Report a side effect with a medicine or medical device
      </p>
      <a
        className="primary-button"
        href="https://yellowcard.mhra.gov.uk/"
        title="Yellow Card"
      >
        Make a report
      </a>
    </div>
  </section>
  <div>
    Child contents
  </div>
</div>
`;
